crates/sargon/src/profile/logic/profile_network/profile_network_details.rs

```diff
@@ -122,6 +122,7 @@ impl AuthorizedPersonaSimple {
             EntitySecurityState::Unsecured { value: uec } => {
                 uec.authentication_signing.is_some()
             }
+            _ => false, // TODO change that
```

Comment:
TODO:
Function that returns authorised persona. Should we `has_auth_signing_key = false` when persona is securified?

Reply:
The working theory is that ALL securified entities will have a ROLA (`CAP26KeyKind::AuthenticatinSigning)` key on them,

Reply:
So `SecuredEntityControl` should hold one optional instance of  `pub authentication_signing: Option<HierarchicalDeterministicFactorInstance>,` like the unsecured one does? This should have nothing to do with the matrix of factor instances?

Reply:
Sorry for slow response.

Rola key is completely outside of MatrixOfFactorInstances.

It is a Wallet construct, not related to the control of any role of an entity.

It would probably not be optional though, since we would set it. Maaaaybe optional since recovery without Profile might dictate it